schemas/com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1

```diff
@@ -0,0 +1,42 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an IP anonymization enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "anon_ip",
+		"format": "jsonschema",
+		"version": "1-0-1"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+			"type": "string"
+		},
+		"name": {
+			"type": "string"
+		},		
+		"enabled": {
+			"type": "boolean"
+		},
+		"parameters": {
+			"type": "object",
+			"properties": {
+				"anonOctets": {
+					"type": "number",
+					"minimum": 1,
+					"maximum": 4
+				},
+        "anonSegments": {
+					"type": "number",
```

Comment:
I don't know why it was `number` in `1-0-0`, but to me it should be an `integer` as e.g. `3.14` would be a clearly invalid value.